src/components/ebay-icon/index.js

```diff
@@ -61,7 +61,7 @@ function getTemplateData(state, input, out) {
         accessibilityAttributes,
         titleId,
         htmlAttributes: processHtmlAttributes(input),
-        classes: ['icon', `icon--${name}`, input.class]
+        classes: input.noSkinClasses ? [input.class] : ['icon', `icon--${name}`, input.class]
```

Comment:
@yomed Is this because we are using inline SVG icons in `<ebay-menu>`? Or the calling component is taking care of the classes?

Reply:
There are a few special case button icons in Skin, where they follow all the rules of regular icons, except without applying the regular classes. In this case, the menu needs to use this for the `expand-btn__icon`.